src/doc/manifest.md

```diff
@@ -451,13 +451,19 @@ To structure your code after you've created the files and folders for your proje
 Files located under `examples` are example uses of the functionality provided by
 the library. When compiled, they are placed in the `target/examples` directory.
 
-They must compile as executables (with a `main()` function) and load in the
```

Comment:
Oh there wasn't anything wrong with it, it just seemed to not flow well having two sentences starting with 'They' back-to-back.